Dockerfile.template

```diff
@@ -20,8 +20,11 @@ RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
 	curl -sSL "https://go.dev/dl/go${GO_VER}.linux-${ARCH}.tar.gz" | sudo tar -xz -C /usr/local/ && \
 	mkdir -p /home/circleci/go/bin
 
-ENV GOPATH /home/circleci/go
-ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+ENV GOPATH="/home/circleci/go"
+ENV PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
+
+# disable telemetry (in a version-independent way)
+go run golang.org/x/telemetry/cmd/gotelemetry@latest off
```

Comment:
```suggestion
RUN go run golang.org/x/telemetry/cmd/gotelemetry@latest off
```

Reply:
Saw this PR would unblock the release (if I understand the CI result on the release PR).

Thanks for looking into getting cimg-go:1.23.0 released!